eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHealthCheck.scala

```diff
@@ -19,7 +20,7 @@ object CassandraHealthCheck {
 
   def of[F[_] : Concurrent : Timer : LogOf](
     session: Resource[F, CassandraSession[F]]
-  ): Resource[F, CassandraHealthCheck[F]] = {
+  )(implicit c: ConsistencyConfig.Read): Resource[F, CassandraHealthCheck[F]] = {
```

Comment:
I really don't like that simplification :)
I'd suggest passing that as `consistencyConfig: ConsistencyConfig.Read` everywhere

Reply:
addressed